### ISSUE TITLE
RECID keyword detected

### ISSUE THREAD

**Issue by patrickTingen** [Riverside-Software/sonar-openedge#767]
I get a false positive when I try to suppres a warning on a RECID statement

```
CASE cColumnName:
    {&_proparse_ prolint-nowarn(recidkeyword)}
    WHEN 'RECID' THEN cColumnValue = STRING(hDataBuffer:RECID).
    WHEN 'ROWID' THEN cColumnValue = STRING(hDataBuffer:ROWID).
    OTHERWISE cColumnValue = hDataBuffer:BUFFER-FIELD(cColumnName):BUFFER-VALUE(iExtentNr).
END CASE.
```
Check DataDigger on the [Riverside demo page](http://sonar.riverside-software.fr/project/issues?branch=develop&id=patrickTingen%3ADataDigger&open=AW2q8ZV6VSds1MSfLiQ8&resolved=false&sinceLeakPeriod=true&types=CODE_SMELL)

**Comment by cverbiest** [comment]
Have you tried with the prolint-nowarn immediately before the statement ?
```
CASE cColumnName:
      WHEN 'RECID' THEN 
      {&_proparse_ prolint-nowarn(recidkeyword)}
      cColumnValue = STRING(hDataBuffer:RECID).
     WHEN 'ROWID' THEN cColumnValue = STRING(hDataBuffer:ROWID).
     OTHERWISE cColumnValue = hDataBuffer:BUFFER-FIELD(cColumnName):BUFFER-VALUE(iExtentNr).
END CASE.
```

**Comment by patrickTingen** [comment]
No, have not tried that, will do.

Patrick Tingen

Op vr 10 jan. 2020 om 13:41 schreef cverbiest <notifications@github.com>:

**Comment by patrickTingen** [comment]
Yes! That solves it, thanks!
